packages/utils/src/sde/shared.ts

```diff
@@ -0,0 +1,126 @@
+import fs from "fs";
+import path from "path";
+import { Bundle } from "@medplum/fhirtypes";
+import { ExtractionBundle, ExtractionSource } from "@metriport/core/external/sde/types";
+
+function getLocalDirectoryPath(directoryName: string) {
+  const localDirectoryPath = path.join(process.cwd(), "runs/sde", directoryName);
+  if (!fs.existsSync(localDirectoryPath)) {
+    fs.mkdirSync(localDirectoryPath, { recursive: true });
+  }
```

Comment:
Don't create the runs folder like this please, it will result in not linking properly to the PHI folder.

See `initRunsFolder`